src/main/java/com/puppycrawl/tools/checkstyle/checks/AbstractTypeAwareCheck.java

```diff
@@ -41,7 +41,6 @@
  * @author Oliver Burn
  * @deprecated Checkstyle is not type aware tool and all Checks derived from this
  *     class are potentially unstable.
- * @noinspection DeprecatedIsStillUsed
```

Comment:
@Vladlis Weren't you the one who added it and enabled it to our config?
https://github.com/checkstyle/checkstyle/commit/5b16c53a8f769c6b25801ff6833ba2922559a01f#diff-57008d74ef07a22dbce6d6d109834cae

Could this be because the level is set to `WARNING`?